module/stylesheets/components/_avatar.scss

```diff
@@ -1,26 +1,27 @@
 .avatar {
   display: inline-flex;
-  height: 50px;
   width: 50px;
+  height: 50px;
```

Comment:
Isn't this a prop?

Reply:
Yes it has to be. Didn't do any feature changes to them. All I did was fixing the linting errors and and some other build issues. Yes we need to update this component. I will fix it.

Reply:
Done